packages/app/src/components/ChatPanel.module.css

```
@@ -0,0 +1,326 @@
+
+.empty {
+  color: #888;
+  font-style: italic;
+  margin: 0;
+}
+
+.turn {
+  padding: 8px 10px;
+  border-radius: 6px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)